Modules/Forum/classes/class.ilForumTopic.php

```diff
@@ -215,13 +215,13 @@ public function reload() : bool
     public function getFirstPostId() : int
     {
         $res = $this->db->queryF(
-            'SELECT * FROM frm_posts_tree WHERE thr_fk = %s AND parent_pos = %s',
-            ['integer', 'integer'],
-            [$this->id, 1]
+            'SELECT * FROM frm_posts_tree WHERE thr_fk = %s AND parent_pos != %s AND depth = %s ORDER BY rgt DESC',
```

Comment:
- Do we stillt need the `parent_pos` condition here?
- Shouldn't we add an `$this->db->setLimit(1);` here as well?